docs/docs/platforms/openshift/OPENSHIFT.md

```diff
@@ -0,0 +1,100 @@
+# Deploy To OpenShift
+
+!!! warning "Support Disclaimer"
+    Helm is a Kubernetes package manager that orchestrates the provisioning of applications onto existing Kubernetes infrastructure. The requirements for this infrastructure are described in [Prerequisites](../../userguide/PREREQUISITES.md). The Kubernetes cluster remains your responsibility; we do not provide direct support for Kubernetes or the underlying hardware it runs on.
+
+    If you have followed our documentation on how to configure the Helm charts, and you're using correctly created components, we will then provide support if you encounter an error with installation after running the `helm install` command.
+
+    Read more about [what we support and what we don’t](../../troubleshooting/SUPPORT_BOUNDARIES.md).
+
+!!! info "Official OpenShift Support"
+    **Starting from version 1.18, Atlassian Data Center Helm charts officially include support for OpenShift**.
+
+## :simple-redhatopenshift: Deploy with a Restricted and Nonroot Security Context
+
+Atlassian Data Center Helm charts are vendor and cloud agnostic, and create objects from standard APIs that [OpenShift](https://www.openshift.com/){.external} fully supports.
+
+However, the default OpenShift **restricted** Security Context Constrain ([SCC](https://docs.openshift.com/container-platform/4.14/authentication/managing-security-context-constraints.html){.external}) requires pods to be run with a UID, and SELinux context that are allocated to the namespace. It means it is not possible to deploy DC Helm charts with the default settings (with pre-defined `securityContext` and containers that run as root). See [FAQs](../OPENSHIFT_FAQ) for typical errors.
+
+!!! warning "ATL_TOMCAT_* Environment Variables"
+    When running as restricted or nonroot SCC, some of `ATL_TOMCAT_*` environment variables passed to the Helm chart in `additionalEnvironmentVariables` will be ignored, because `server.xml` and  `seraph-config.xml` are mounted as a ConfigMaps rather than generated in the container entrypoint from Jinja templates. Both `<product>.tomcatConfig` and `<product>.seraphConfig` have a number of properties which you can override if necessary. Look at `tomcatConfig` stanza in the chart's values.yaml for more details.
+
+=== "Restricted SCC (:octicons-thumbsup-16: Recommended)"
+    !!! info "Restricted SCC"        
+        Restricted SCC denies access to all host features and requires
+        pods to be run with a UID, and SELinux context that are allocated to the namespace.
+    
+    Running containers in OpenShift with a restricted SCC is the most secure approach. To be able to successfully deploy Atlassian DC Helm charts with a restricted SCC, set `openshift.runWithRestrictedSCC` to true in Helm values file:
+
```

Comment:
There's a missing "the". Change the last phrase to "to true in the Helm values file:".